### PR TITLE
dbaas-mycnf packaging in the build step.

### PR DESCRIPTION
**Pull request by hub-cap**
- Added the package repo creation
- Added apache2 vhost on 8080 (horizon took 80)
- Added a build of dbaas-mycnf and insert into pkg repo
- Added a file for the guest instance to be able to read from repo

### CODE REVIEW

File: scripts/redstack.rc [hunk 1]
```diff
@@ -31,6 +31,10 @@ PATH_TESTS_CONF="$PATH_INTEGRATION_CONF/tests.conf"
 
 PATH_ENV_CONF="$PATH_INTEGRATION_CONF/env.rc"
 
+# Package repo directory
+PACKAGES_DIR=/var/lib/packages/debian/
+BUILD_DIR=/tmp/build/
+
 # Passwors used by devstack.
```
Comment (cp16net) [inline scripts/redstack.rc]: +d

Comment (hub-cap) [inline scripts/redstack.rc]: Haha nice. Illl add that fix to my pr.

**Comment by rnirmal** [comment]
+1

**Comment by TimSimpsonR** [comment]
+1

**Comment by ed-** [comment]
+1

**Pull request merged**